rulebook.tex

```diff
@@ -763,315 +977,132 @@ \subsubsection{Markers}
+Depending on the game mode, either the exploration or production phase follows.
 
-For each game, one CS and one RS of each team will be chosen randomly
-and swapped with the corresponding machine of the other team on the
-non-primary half --- that is two machines in total per team. The CS and
-RS will be swapped with the symmetrically positioned machine of the
-same type of the other team. For consecutive games, the changes to the
-machine to zone assignment may be limited by the referee box in order
-to reduce the time needed to implement the layout during the setup
-phase.
-
-\subsubsection{MPS --- During Exploration Phase}
-\label{sec:production-machines-exp}
-During the exploration phase the robots of each team must explore the
-unknown factory environment to identify where the different MPS are
-located and in which rotations they are positioned.
-
-The Referee Box assigns each machine to a team and a zone
-(\reffig{fig:competition-area}) at the start of the exploration
-phase,\footnote{Since 2017 there are no more fixed stations.}
-The zones and orientations (in steps of 45$^\circ$) are randomly determined,
-and will not be communicated until the production phase. For details about the
-Exploration Phase confer~\refsec{sec:exploration-phase}.
-
-The signal light of the MPS will show a yellow light until a report regarding
-this machine is sent.  If the wrong MPS zone is sent, the light will start red
-flashing.  If only the MPS zone is sent, the green light will be turn on.  If
-the zone and rotation is correctly reported, the green light will be flashing
+\subsection{Exploration Phase}
+\label{sec:exploration-phase}
+During the exploration phase the robots of each team have \SI{3}{\minute}
+to roam the environment and report their (and only their) machines
+to the \ac{refbox}.
+Correctly reported machines will be rewarded with points.
+Reports are issued to the refbox using a \emph{MachineReport message}.
+For each machine,identified by its name, both its zone and its orientation
+(in degrees) can be explored.
+Both properties can be submitted at the same time in one message or be split up
+into multiple messages.
+The refbox accepts only the first submitted value for each property.
+For a detailed description of the message please refer to the \ac{refbox}
+wiki\footnote{\url{https://github.com/robocup-logistics/rcll-refbox/wiki/Communication-Protocol}}. % chktex ignore-long-line
+The scoring is shown in \reftab{tab:scoring}.
+A minimum of zero points will be accounted for the exploration phase.
+
+The signal light of the \ac{MPS} will show a yellow light
+until a report regarding this machine is sent.
+If the wrong \ac{MPS} zone is sent, the light will start red
+flashing.  If only the \ac{MPS} zone is sent, the green light will be turn on.
+If the zone and rotation is correctly reported, the green light will be flashing
 and if the zone is correct but the rotation wrongly reported the red and green
 light will be on.
```

Comment:
A table with Positon Ok Rotation OK -> Green light and so on is more easy to read that very longs sentences.

Reply:
Good point, i will do that for my PR regarding unified exploration and production phase as i will need to change the color codes anyways to avoid overlaps between exploration phase codes and production phase codes.